packages/dnb-eufemia/src/shared/colors.ts

```diff
@@ -0,0 +1,463 @@
+/**
+ * DNB UI / Sbanken / DNB Eiendom colors
+ */
+
+type ColorEntry = {
+  name: string
+  cssVariable: string
+  hex: string
+  rgb: string
+}
+
+function createColorEntry(
+  name: string,
+  hex: string,
+  rgb: string,
+  cssVariable: string
+): ColorEntry {
+  return { name, hex, rgb, cssVariable }
+}
+
+type UiColor = {
+  oceanGreen: ColorEntry
+  emeraldGreen: ColorEntry
+  seaGreen: ColorEntry
+  mintGreen: ColorEntry
+  summerGreen: ColorEntry
+  accentYellow: ColorEntry
+  indigo: ColorEntry
+  violet: ColorEntry
+  skyBlue: ColorEntry
+  lavender: ColorEntry
+  sandYellow: ColorEntry
+  pistachio: ColorEntry
+  seaGreen30: ColorEntry
+  mintGreen50: ColorEntry
+  mintGreen25: ColorEntry
+  mintGreen12: ColorEntry
+  accentYellow30: ColorEntry
+  signalOrange: ColorEntry
+  fireRed: ColorEntry
+  fireRed8: ColorEntry
+  successGreen: ColorEntry
+  black: ColorEntry
+  black80: ColorEntry
+  black55: ColorEntry
+  black20: ColorEntry
+  black8: ColorEntry
+  black3: ColorEntry
+  white: ColorEntry
+}
+
+type SbankenColor = {
+  profilePurple: ColorEntry
+  profilePurpleAlternative: ColorEntry
+  profileGreen: ColorEntry
+  profileRed: ColorEntry
+  profileMagenta: ColorEntry
+  profileOrange: ColorEntry
+  profileYellowDark: ColorEntry
+  profileYellow: ColorEntry
+  profileGreenDark3: ColorEntry
+  profileGreenDark2: ColorEntry
+  profileViolet: ColorEntry
+  profileVioletLight: ColorEntry
+  profileBlueDark2: ColorEntry
+  profileBlueDark: ColorEntry
+  profileBlue: ColorEntry
+  uxBlack: ColorEntry
+  uxText: ColorEntry
+  uxGrayDark3: ColorEntry
+  uxGrayDark2: ColorEntry
+  uxGrayDark: ColorEntry
+  uxGray: ColorEntry
+  uxGrayLight: ColorEntry
+  uxGrayLight2: ColorEntry
+  uxGrayLight3: ColorEntry
+  uxGrayDark3Neutral: ColorEntry
+  uxGrayDark2Neutral: ColorEntry
+  uxGrayDarkNeutral: ColorEntry
+  uxGrayNeutral: ColorEntry
+}
+
+type EiendomColor = {
+  emeraldGreen50: ColorEntry
+  emeraldGreen25: ColorEntry
+  emeraldGreen10: ColorEntry
+  mintGreen12: ColorEntry
+}
+
+const ui: UiColor = {
+  oceanGreen: createColorEntry(
```

Comment:
```suggestion
    oceanGreen: {
    name: 'Ocean green',
    hex: '#00343E',
    rgb: 'rgb(0, 52, 62)',
    cssVariable: 'var(--color-ocean-green)',
  },
```

Could also consider this instead.